frontend-web/src/app/[locale]/map-interactive/page.tsx

```diff
@@ -0,0 +1,16 @@
+'use client';
+
+import dynamic from 'next/dynamic';
+import React, { ReactElement } from 'react';
+
+const Map = dynamic(() => import('../../components/sections/Map'), {
+  ssr: false,
```

Comment:
tu peux aussi rajouter un loader pendant le chargement de la carte

Par exemple : 
```suggestion
  ssr: false,
  loading: () => <p>Chargement de la carte...</p>,
```